src/patterns/service-unavailable-pages/link-to-another-service/index.njk

```diff
@@ -0,0 +1,20 @@
+---
+layout: layout-example.njk
+---
+
+<div class="govuk-width-container">
+
+  <div class="govuk-main-wrapper">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+
+        <h1 class="govuk-heading-xl">Service unavailable</h1>
+        <p class="govuk-body-l">You will be able to use the service from 9am on Monday 19 November 2018.</p>
+        <p class="govuk-body">If you need to make a payment and know how much you need to pay, <a class="govuk-link" href="#">use the payment service</a>.</p>
```

Comment:
Change the line to:

`<p class="govuk-body">You can <a class="govuk-link" href="#">change other VAT details</a>.</p>`